### DISCUSSION TITLE
-----

### DISCUSSION THREAD

**Discussion by cyberrrr** [Rapptz/discord.py#6323] [General]
----

**Comment by 15696** [comment]
[`bot.user.blocked`](https://discordpy.readthedocs.io/en/latest/api.html#discord.ClientUser.blocked) should work.

**Reply by timedoutuk** [reply]
This only works for self-bots though

**Reply by 15696** [reply]
> This only works for self-bots though

I am aware.

**Comment by Rapptz** [comment]
Bot users cannot block people.

**Reply by 15696** [reply]
I think they were referring to selfbots. Are we allowed to help with them here?

**Reply by tk421bsod** [reply]
> 
> I think they were referring to selfbots. Are we allowed to help with them here?

Probably not, as selfbots are against Discord's TOS.
